Yoast/javascript
# Add specs for the French suffix -ition and edit the goldStandard list

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Adds specs for the French suffix -ition (in singular and plural) and edits the goldStandardStems file in oder to get the correct stems out of the nouns ending in -ition and -itions.

* [Yoast SEO Premium] Adds a fix for the nouns ending in -ition and itions (e.g., _supposition, suppositions_)so that they can be recognised and matched with the corresponding verb and/or adjective (e.g., _supposer_).

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* In yoastseo: Run yarn test and make sure all current specs pass
* In Yoast SEO Premium: Add nouns ending in -ition to the keyphrase and to the title/text/meta description and check if the corresponding verbs or adjectives are recognised

## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LIN-408
